source/common/http/filter/gzip_filter.cc

```diff
@@ -190,7 +190,7 @@ bool GzipFilter::isAcceptEncodingAllowed(HeaderMap& headers) const {
     return is_wildcard;
   }
 
-  return true;
+  return false;
```

Comment:
i would add a comment explaining the implication here (e.g. "default off if no header present")

Reply:
👍